unused/downloads.md

```diff
@@ -26,7 +26,7 @@ An [interactive version of the Guidelines](http://music-encoding.org/support/gui
 
 ## Sample encodings
 
-[![sampleEncoding](http://music-encoding.org/wp-content/uploads/2015/04/sampleEncoding-300x176.jpg)](http://music-encoding.org/wp-content/uploads/2015/04/sampleEncoding.jpg)
```

Comment:
As this is outdated and not available on the website, I wonder if we even try to fix this link. Shouldn't we completely remove the unused files, @kepper?

Reply:
That was my question exactly, also asking it in #189 but since Stefan provided a link I thought it might not hurt to include it. However, I wouldn't mind changing it.

Reply:
My impression is that the complete `unused` folder was kept for historical reasons. If we keep it, we should also fix the link and discuss the need (or not) of the `unused` folder in another issue/PR.